transcendence-app/src/shared/enums/game-mode.enum.ts

```diff
@@ -0,0 +1,6 @@
+export enum GameMode {
+  training = 'training',
+  classic = 'classic',
+  quick = 'quick',
+  hidden = 'hidden',
```

Comment:
suggestion: I think, for now, we could add only the 'classic' mode and add others when we implement them.